README.md

```diff
@@ -7,20 +7,23 @@ for prototyping etc.
 This Github fork of Skript is based on Mirreski's improvements which was built
 on Njol's original Skript.
 
-## Supported Versions
+## Requirements
 Skript requires **Spigot** to work. You heard it right, Bukkit does *not* work.
 **Paper**, which is a fork of Spigot, is recommended; it is required for some
 parts of Skript to be available.
 
-Skript supports **latest releases** of Minecraft 1.9, 1.10, 1.11, 1.12 and 1.13.
-For example, this means that 1.12.2 is supported, but 1.12.1 is *not*.
-Testing with all minor Minecraft releases is not feasible for us.
+Skript supports only the **latest** patch versions of Minecraft 1.9, 1.10, 1.11, 1.12 and 1.13.
+This means that 1.12.2 is supported, but 1.12.1 is *not*.

```

Comment:
For example should be kept. This applies to ALL supported versions, not just 1.12